NachoSpigot-Server/src/main/java/net/minecraft/server/PlayerConnection.java

```diff
@@ -1493,10 +1495,11 @@ public void a(PacketPlayInUseEntity packetplayinuseentity) {
             double d0 = 36.0D;
 
             if (!flag) {
-                d0 = 9.0D;
+                // Increase the no player-player vision maximum reach
+                d0 = (NachoConfig.enableImprovedHitReg) ? 12.75D : 9.0D;
```

Comment:
I can't see what code is above this line, but does this mess with anticheats or not? Might need to be tested.

Reply:
The server I run utilizes the Vulcan anti-cheat and I haven't run into any issues yet, although it might actually break some things depending on the anti-cheat. I personally haven't checked to see if the checks have been altered in any way, so I can't say for sure. But I can definitely say that my changes have not caused any false positives yet.